tools/topic_tools/package.xml

```diff
@@ -18,6 +18,8 @@
   <buildtool_depend version_gte="0.5.78">catkin</buildtool_depend>
 
   <build_depend>cpp_common</build_depend>
+  <build_depend>geometry_msgs</build_depend>
```

Comment:
This would introduce a circular dependendcy between this repo and the `common_msgs` repo. That would make the first time release of these repositories problematic since both need each other.

Maybe moving this functionality into a separate repository would be better.

Reply:
@dirk-thomas Thank you for the feedback! The root problem is there is no message which has Header field in this repo which is need for testing. Would it be nice if I create a new package (let the name be `test_topic_tools`) in `[test](https://github.com/ros/ros_comm/tree/melodic-devel/test)` directory and move all tests for this package into `test_topic_tools` and also generate a dummy message with header field?

Reply:
Sounds good to me :+1:

Reply:
@furushchev Friendly ping.

Reply:
@dirk-thomas Thanks for pinging. Just now updated this PR according to the discussion above.

Reply:
@dirk-thomas If it would be better to move all existing test codes in `topic_tools` to `test_topic_tools`, I'll do it.